pootle/apps/pootle_store/models.py

```diff
@@ -356,6 +359,11 @@ class Unit(models.Model, base.TranslationUnit):
                                     db_index=True, related_name='reviewed')
     reviewed_on = models.DateTimeField(db_index=True, null=True)
 
+    # this is calculated from virtualfolders if installed and linked
+    priority = models.FloatField(
+        db_index=True, default=1,
+        validators=[MinValueValidator(0.9)])
```

Comment:
Actually `0.0` is not even valid: https://github.com/phlax/pootle/blob/unit_priority/pootle/apps/virtualfolder/models.py#L202-L203